library/src/main/java/io/constructor/data/model/search/Result.kt

```diff
@@ -0,0 +1,12 @@
+package io.constructor.data.model.search
+
+import com.squareup.moshi.Json
+import io.constructor.data.model.Group
+import io.constructor.data.model.ProductData
+
+data class Result(val description: String?, val id: String,
```

Comment:
We should discuss typing tomorrow.  Some of these properties are universal and some are not :(

Reply:
not sure if I get this one